src/pages/index.js

```diff
@@ -398,7 +407,13 @@ const Index = ({ data: initialData }) => {
           ) : null}
           <div className={classes.fullContainer}>
             <div className={classes.topContainer}>
+              <Button className={classes.feedbackLink}>
+                <Link to={surveyLink}>
```

Comment:
Move LInk outside of the button. Currently, you can click the button (around the text) and go nowhere.